_posts/2017-04-06-gundam-origins-vol-1.md

```diff
@@ -0,0 +1,99 @@
+title: "Mobile Suit Gundam The Origin - Vol 1 Activation"
+description: "Giant robots in space. How could you not love it?"
+tags: [review]
+modified: 2017-04-06
+author: Chao Xian
+image:
+  feature: comic/gundam-origin/header.jpg
+  credit: Yoshikazu Yasuhiko
+  creditlink: http://www.vertical-inc.com/books/gundam.html
+---
+
+They say you should never judge a book by its cover but wow look at this one. It stirs a wistful excitement in me rooted from a time long ago and far, far away.
+
+[![The cover to Mobile Suit Gundam Origin vol 1 Activation]({{site.url}}/images/comic/gundam-origin/cover.jpg)]({{site.url}}/images/comic/gundam-origin/cover.jpg)
+
+Be prepared for a long ramble about what I patchily know of Gundam. It is both comfortingly familiar to me and also a mystery. The familiarity stems from the whole brand being so iconic, yet a mystery because I never really delved into it. But before I bore you with my love of giant-robo mecha suits, let's hear from someone who doesn't really like giant robots.
+
+## Jake and the giant robots
+
+Let's start with what he didn't get on with. He found it a bit of a confusing mess. Although he questioned whether he just wasn't used to the genre, and was unable to fill in space. He also sometimes struggled distinguish the spaceships from the robots.
+
+Jake couldn't work out what was going on in the space battles. The confusion combined with Jake's disengagement with the characters led to him admitting he didn't really pay much attention to it. He was rather disinterested in the Battlestar Galactica plot.
+
+What he did like was the art. Particularly the colour segments, calling it arresting. He wished it was all in colour. Despite not liking most of the characters, he did single out Char Aznable - the dashing arch-nemesis of our rather bland protagonist. He found it amusing how hard-assed Sayla Mass gasps at Char when he removes his helmet to reveal his handsome visage, noting it as superb.
+
+Jake conceded that the book looked dead cool though.
+
+---
+
+:star::star:
+
+> Giant robots are not for me. There were no farting sharks.
+>
+> -- <cite>[@tygertale](https://twitter.com/tygertale)</cite>
+
+
+## Confused Dan
+
+Dan echoed much of the same points as Jake, citing his loss of sense of scale in the opening scene. He didn't realise the Zaku were giant robots at first. He was also unsure who were the goodies and the baddies.
+
+He found some of the art absolutely gorgeous and looked absolutely amazing, but Dan wanted to understand what was going on.
+
+He supposed that it could be good as a dark military story and quite hard hitting war story, and in some cases it was, but then red ferret with his super cape turns up (I'm assuming he means Char) and Dan found that he couldn't stop laughing.
+
+Some not particularly spectacular to truly spectacular - it feels like it's of its time although it's like Star Wars it drawn as the future and cool but you can feel that it was done in the late 70s/80s
```

Comment:
Ya know, that's lifted from my notes where I was basically transcribing what people said. I stopped editing at one point 😂